lib/redis/config.rb

```diff
@@ -0,0 +1,48 @@
+module Redis::Config
+  DEFAULT_SENTINEL_PORT = '26379'.freeze
+  SIDEKIQ_SIZE = 25
+
+  class << self
+    def app
+      return BASE_CONFIG if const_defined? 'BASE_CONFIG'
+
+      reload_config
+    end
+
+    def sidekiq
+      config = begin
+        if const_defined? 'BASE_CONFIG'
+          BASE_CONFIG
+        else
+          reload_config
+        end
+      end
+      config.merge(size: SIDEKIQ_SIZE)
+    end
+
+    def reload_config
```

Comment:
@sojan-official I added this `reload_config` method to help me change and test configuration under different ENV variable scenarios. Though, I am not feeling very confident of this implementation. Can you suggest a better way?

Reply:
looks good to me at the first glance. 
@sony-mathew  could you take a look ?

Reply:
We can definitely refactor this. But if develop is broken right now and this fixes it, should we merge this and then refactor?

Reply:
@sony-mathew Redis Sentinel feature which was introduced recently is broken, previous redis configuration is okay for app and sidekiq.

Reply:
@monarch0111  @sony-mathew  I think we can merge this in and do further refactoring in subsequent PRs if required.